build/Uno.WinUI.Svg.nuspec

```diff
@@ -0,0 +1,140 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata minClientVersion="5.0">
+		<id>Uno.WinUI.Svg</id>
+		<version>0.0.1</version>
+		<title>Uno.WinUI.Svg</title>
+		<authors>nventive</authors>
+		<owners>nventive</owners>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<projectUrl>https://platform.uno/</projectUrl>
+		<license type="expression">Apache-2.0</license>
+		<icon>uno.png</icon>
+		<description>
+			Support for Svg on Uno.WinUI.
+		</description>
+		<copyright>Copyright (C) 2015-2022 uno platform inc. - all rights reserved</copyright>
+		<repository type="git" url="https://github.com/unoplatform/uno.git" branch="$branch$" commit="$commitid$" />
+
+		<dependencies>
+
+			<group targetFramework="netstandard2.0">
+				<dependency id="Uno.WinUI" version="to-be-set-by-ci" />
+				<dependency id="System.Json" version="4.7.1" />
+			</group>
+
+			<group targetFramework="net7.0">
```

Comment:
`net6` is supported by the `netstandard2.0` target in this package, so we're good to go :) The package itself does not need anything net6 or net7 specific, but it will likely use some of the newer generation features in the future.